docs/docs/architecture.md

```diff
@@ -107,7 +107,8 @@ not correct, this will be reported to the client by Intel's servers as part of v
 the remote attestation provided by the host.
 
 **Cloud provider.** A cloud provider needs to support SGX for Conclave to be usable. They may operate their own
-provisioning servers that take over from Intel's. Instructions for using Conclave on [Microsoft Azure](azure.md) are available.
+provisioning servers that take over from Intel's. Instructions for using Conclave on
+[Microsoft Azure](machine-setup.md) are available.
```

Comment:
I would move elsewhere the actual link. When reading it, I somehow expected the `Microsoft Azure` link to point me to Microsoft Azure. I personally prefer something lile
`Instructions for using Conclave on Microsoft Azure [are available here](machine-setup.md)` or similar

Reply:
Done.